Classes/Issues/Labeled/IssueLabeledSectionController.swift

```diff
@@ -9,13 +9,17 @@
 import Foundation
 import IGListKit
 
-final class IssueLabeledSectionController: ListGenericSectionController<IssueLabeledModel> {
+
+
+final class IssueLabeledSectionController: ListGenericSectionController<IssueLabeledModel>, MarkdownStyledTextViewDelegate {
 
     private let issueModel: IssueDetailsModel
+    private weak var tapDelegate: IssueLabelTapSectionControllerDelegate?
 
-    init(issueModel: IssueDetailsModel) {
+    init(issueModel: IssueDetailsModel, tapDelegate: IssueLabelTapSectionControllerDelegate) {
         self.issueModel = issueModel
         super.init()
+        self.tapDelegate = tapDelegate
```

Comment:
_Technically_ should be set before init, but getting away w/ it since its `weak`